src/main/java/roomescape/auth/application/CookieUtils.java

```diff
@@ -0,0 +1,36 @@
+package roomescape.auth.application;
+
+import jakarta.servlet.http.Cookie;
+import org.springframework.stereotype.Component;
+import roomescape.exception.UnauthorizedException;
+
+import java.util.Optional;
+
+@Component
+public class CookieUtils {
+    public Cookie createCookie(String name, String value) {
+        Cookie cookie = new Cookie(name, value);
+        cookie.setHttpOnly(true);
+        cookie.setPath("/");
+        return cookie;
+    }
+
+    public Optional<Cookie> getOneCookieByName(Cookie[] cookies, String name) {
```

Comment:
```suggestion
    public Optional<Cookie> getCookieByName(Cookie[] cookies, String name) {
```

꼭 필요하지 않은 표현은 이름에서 제거해봐도 좋을 것 같아요.